src/meta/src/hummock/manager/mod.rs

```diff
@@ -102,6 +102,7 @@ pub struct HummockManager<S: MetaStore> {
     // CompactionGroupId
     compaction_request_channel: parking_lot::RwLock<Option<CompactionRequestChannelRef>>,
     compaction_resume_notifier: parking_lot::RwLock<Option<Arc<Notify>>>,
+    compaction_tasks_to_cancel: tokio::sync::Mutex<Vec<HummockCompactionTaskId>>,
```

Comment:
Can we use a sync lock (e.g. `parking_lot::Mutex`) instead of an async tokio lock here since we don't need to hold the lock across await point?

Reply:
> Can we use a sync lock (e.g. `parking_lot::Mutex`) instead of an async tokio lock here since we don't need to hold the lock across await point?

fixed